### ISSUE TITLE
V1 or V2 Wink API

### ISSUE THREAD

**Issue by taconugget** [KraigM/homebridge-wink#55]
Does this plug-in use v1 or v2 of the wink API? I just got my api login but it doesn't work. I suspect maybe I need to ask for v1? The 

Thanks

**Comment by thechrisreese** [comment]
V1

**Comment by Sturgeon637** [comment]
Just got this from Wink

Hi Joe,

We are no longer able to generate v1 API credentials anymore and only provide v2. I apologize for any inconvenience this may cause. Let me know if you have any other questions or concerns. Thanks!


Cheers, 
Victoria

**Comment by thechrisreese** [comment]
Correct. They only give out V2. Find the Android credentials (google them) and use those.

**Comment by taconugget** [comment]
Just out of curiosity is there any plan to update the API in the plugin?

**Comment by thechrisreese** [comment]
No one knows. The Dev posted a while back he was working on it. But that was months ago.

**Comment by FJRGuy** [comment]
Doh! Just requested my credentials. Guess they won't work. 
I'd take a shot at updating the plugin for the API, but pretty sure the results would be a dismal failure. (I'm not much of a coder. ;-)

**Comment by mahodder** [comment]
These generic v1 keys still work for me:

> "client_id": "quirky_wink_android_app",
> "client_secret": "e749124ad386a5a35c0ab554a4f2c045",

**Comment by FJRGuy** [comment]
Awesome! I’ll give them a try. Thanks!

**Comment by jpartain89** [comment]
The generic V1 keys are what works. So, the specific instructions posted here for the API info needs to formally be changed to @mahodder info.
